user.txt

```diff
@@ -0,0 +1 @@
+i = 0;
```

Comment:
it would be better to use "y" instead of "i".

Reply:
@EMS1296 i is good because it is conventional, we can discuss this further if you'd like